README.md

```diff
@@ -19,17 +17,19 @@ However, it does not provide a SSH export as the DnD Magento connector bundle do
 
 | EnhancedConnectorBundle | Akeneo PIM Community Edition |
 |:-----------------------:|:----------------------------:|
+| v1.6.*                  | v1.6.*                       |
```

Comment:
1.3.* for 1.6.* in fact because we can't align any bundles (for instance custom entity bundle)